examples/example_sdl_opengl3/Makefile

```diff
@@ -61,9 +78,16 @@ CXXFLAGS += -I../libs/gl3w -DIMGUI_IMPL_OPENGL_LOADER_GL3W
 
 ifeq ($(UNAME_S), Linux) #LINUX
 	ECHO_MESSAGE = "Linux"
-	LIBS += -lGL -ldl `sdl2-config --libs`
 
+ifndef RPI
+	LIBS += -lGL -ldl `sdl2-config --libs`
 	CXXFLAGS += `sdl2-config --cflags`
+else
+	ECHO_MESSAGE = "Raspberry Pi"
+	LIBS += `PKG_CONFIG_PATH=/opt/vc/lib/pkgconfig pkg-config --libs bcm_host brcmegl brcmglesv2` -ldl `sdl2-config --libs`
```

Comment:
This is not required on a Pi4/3 running with the new OpenGL KMS driver.
Standard pkg-config is just enough.

`pkg-config glesv2 --libs`

It is required if one really wants the legacy driver.

Reply:
I do not know much about the the KMS driver.
Do you have any input on what are differences between the legacy vs. KMS driver?
Does the KMS driver postulate use of X11?

Reply:
@audetto
> This is not required on a Pi4/3 running with the new OpenGL KMS driver.
> Standard pkg-config is just enough.
> 
> `pkg-config glesv2 --libs`
> 
> It is required if one really wants the legacy driver.

I am doing this right now, and getting 60FPS, but seeing 100% usage of all 4 cores.   Not really sure why.    My earlier pull request does not work on Rpi4 because I am not sure the legacy is even available.

Reply:
@hippyau 
> I am doing this right now

Which Pi? Which KMS? It is a pain, but without this info, one cannot understand.

Anyway, this PR, on a **Pi4** using **Full KMS**, gives `* failed to add service - already in use?`

On the other hand, if I do this
```
	LIBS += -lGL -ldl `sdl2-config --libs`
	CXXFLAGS += `sdl2-config --cflags`
```

It works. Maximised (**not** fullscreen), about 10% CPU, 60 FPS.

Reply:
Sorry,  was talking about Pi4, no X.... 

I have just seen the weirdest thing...  
running `example_sdl_opengl3` gets 100% CPU and ~40 FPS...
running `sudo example_sdl_opengl3` gets 60FPS solid and 10% CPU.